src/Config.php

```diff
@@ -51,6 +30,27 @@ public function __construct($config)
     /** @var string Password to use for authentication */
     public $password;
 
+    /** @var bool Whether to use SSL */
+    public $useSsl;
```

Comment:
Unfortunately, the SSL options are specific to the database driver used, e.g. MySQL, PostgreSQL, MSSQL all have different options. So we cannot simply define them here. We could do the following:

1. Store options that are not represented by public properties in an extra array and implement `__get`, `__set`. The `property_exists()` call can be removed then.
2. Concrete `Config` implementations for the database drivers. This would require quite some rework so I opt for option 1.

Reply:
I've kept the `property_exists()` call to determine, if the values should be saved in the dedicated property, or the array.

Reply:
Also, `useSsl` should not be a public property as it is specific to the database driver used.